syntax = "proto2";

//import "dccl/option_extensions.proto";

//package jaiabot.sensor.protobuf;

message AtlasScientificOEMDO
{
    //option (dccl.msg) = {
    //    unit_system: "si"
    //};

    // notional - add units and additional fields as needed
    optional double dissolved_oxygen = 1;
}



 